CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [UNRELEASED] neptune 1.6.2
+### Changes
```

Comment:
Can we make a newline between the header and the `### Changes`? I'm really sorry for asking for that :D